### DISCUSSION TITLE
support for markdown-it's typographer?

### DISCUSSION THREAD

**Discussion by schpet** [markdoc/markdoc#516] [Q&A] [answered]
markdown-it supports a 'typographer' option, can i use this with markdoc?

[markdown-it's demo](https://markdown-it.github.io) has a 'typographer' checkbox to enable this, it will do things like replace regular quotes with curly quotes.

**Comment by matv-stripe** [comment, marked as answer]
Yup!


```js
const tokenizer = new Markdoc.Tokenizer({
  typographer: true,
  allowIndentation: true,
});
const ast = Markdoc.parse(tokenizer.tokenize(source));
const content = Markdoc.transform(ast, markdocConfig);
```

**Reply by schpet** [reply]
cool thanks! hoping to get this into astro– https://github.com/withastro/astro/pull/11450
